pkg/porter/testdata/show/expected-output.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "test",
+  "created": "2020-04-18T01:02:03.000000004Z",
+  "modified": "2020-04-18T01:02:03.000000004Z",
+  "bundle": "porter-hello",
+  "version": "0.1.0",
+  "action": "upgrade",
+  "status": "running",
+  "parameters": [
+    {
+      "name": "logLevel",
+      "type": "integer",
+      "sensitive": false,
+      "value": 3
+    },
+    {
+      "name": "token",
+      "type": "unknown",
+      "sensitive": false,
+      "value": "top-secret"
+    }
+  ],
+  "outputs": [
+    {
+      "name": "bar",
+      "type": "string",
+      "sensitive": false,
+      "value": "bar-output"
+    },
+    {
+      "name": "foo",
+      "type": "string",
+      "sensitive": true,
+      "value": "foo-output"
+    }
+  ],
+  "history": [
+    {
+      "claimID": "1",
+      "bundle": "porter-hello",
+      "version": "0.1.0",
+      "action": "install",
+      "parameters": {
+        "logLevel": 5,
+        "token": "top-secret"
+      },
+      "timestamp": "2020-04-18T01:02:03.000000004Z",
+      "status": "succeeded",
+      "hasLogs": "false"
+    },
+    {
+      "claimID": "3",
```

Comment:
Just found this curious: a jump from 1 to 3 for claimID...  A quirk of the `CreateClaim` helper?

Reply:
it's a quirk of how the test generates repeated id's for claims and results. I was lazy and the number didn't really matter, it just had to be consistent between tests. 😀